src/gui/entry/EntryModel.h

```diff
@@ -85,6 +87,7 @@ private slots:
     void severConnections();
     void makeConnections(const Group* group);
 
+    bool m_colorVisible = true;
```

Comment:
This variable (and corresponding function) is ambiguous. Should add the word "Background" in both.